app/views/address/ChooseAddressView.scala.html

```diff
@@ -33,7 +33,7 @@
     @formHelper(action = routes.ChooseAddressController.onSubmit(waypoints, addressJourneyType), 'autoComplete -> "off") {
 
         @if(form.errors.nonEmpty) {
-            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value-0")))
```

Comment:
are we sure about this ?

Reply:
@sqinnydave could you advise?

Reply:
without this the link in error summary when clicked doesnt go to the first radio button as it should ( the id's on radios take the -0, -1, -2 formatting, not underscore.)